src/layouts/App.scss

```diff
@@ -24,6 +24,10 @@ button:focus {
   outline: none;
 }
 
+input:-webkit-autofill {
```

Comment:
How will making this effectively global affect other inputs where we don't to alter the behavior?

Reply:
If we decide to disable the autocomplete background there I it's suppose to be across all Alchemy consistently.

Reply:
No I don't think so.  The other cases across Alchemy do not have the custom background that the Search inputs have.  Thus I suggest that the default behavior is desirable in those cases, no reason to change it.